hyperparameter_tuning/rapids_bring_your_own/rapids_sagemaker_hpo.ipynb

```diff
@@ -0,0 +1,1819 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With all these powerful tools at our disposal, every data scientist should feel empowered to uplevel their model before serving it to the world!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sagemaker\n",
+    "from helper_functions import *\n",
+    "working_directory = get_notebook_path()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src='img/hpo.png'>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<span style=\"color:#8735fb; font-size:22pt\"> **Key Choices** </span>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's go ahead and choose the configuration options for our HPO run.\n",
+    "\n",
+    "Note that we've set the configuration defaults to values that are on the low end, however you are welcome to scale them up. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<span style=\"color:#8735fb; font-size:18pt\"> [ Dataset and S3 Bucket ] </span>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We target a large real-world structured dataset of flight logs for US airlines ( published monthly since 1987 by the Bureau of Transportation [dataset link](https://www.transtats.bts.gov/DatabaseInfo.asp?DB_ID=120&DB_URL=), additional details in <a href='#dataset'>Section 1.1</a>). \n",
+    "\n",
+    "We offer several sizes of this dataset in <a href='https://parquet.apache.org/'>parquet</a> (compressed column storage) format: `1_year` (2019, 7.2M flights), `3_year` (2016-2019, 18M flights) or `10_year` (2009-2019, 63M flights). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset_directory = '1_year'\n",
+    "\n",
+    "assert( dataset_directory in [ '1_year', '3_year', '10_year'] )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For demo purposes we host these pre-built datasets in a public S3 bucket, you are of course welcome to copy the data into the corresponding region of your compute, or host your own dataset in a bucket of your choosing."
```

Comment:
Performance happens when querying from a S3 bucket (has to be in the same region as the compute per SageMaker requirements). Added clarity around the region options as well as a validation check, see below in the comment where you experienced an error because of a region mis-match.